src/app/legacy/containers/EpisodeList/Image.jsx

```diff
@@ -30,16 +30,31 @@ const Wrapper = styled.div`
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     ${({ dir }) =>
       `margin-${dir === 'ltr' ? 'right' : 'left'}: ${GEL_SPACING_DBL};`}
-    ${({ isLite }) => `width: ${isLite ? '7.5rem' : '14.375rem'};`}
+    width: 14.375rem;
   }
 `;
 
+const LitePlayWrapper = withEpisodeContext(styled.div`
+  padding: ${GEL_SPACING_HLF};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING};
+  }
+  svg {
+    margin: 0 0 1px 0;
```

Comment:
Not quite sure what this element is, and if it matters that we have a fixed value rather than relative

Reply:
A slight margin is being given to the SVG on onDemandTV pages, I've changed this to a relative value now, as it should change based on font size


https://www.bbc.com/hausa/bbc_hausa_tv/tv_programmes/w13xttn0.lite
https://www.bbc.com/urdu/bbc_urdu_tv/tv_programmes/w13xttn1.lite